src/pages/listing/ListingPage.tsx

```diff
@@ -9,7 +9,7 @@ import BookingBar from './BookingBar';
 import { Listing } from 'networking/listings';
 
 const ListingPage = (listing: Listing) => (
-  <>
+  <Fade>
```

Comment:
sometimes it's not good to use a `Fade` effect. I think it's fine here.

ie, if we want the page to load immediately, like the signup/booking pages. Anything slowing the user down to conversion is not good. cc @kcvan